client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/style.scss

```diff
@@ -191,44 +191,57 @@
 	pointer-events: none;
 }
 
-.launchpad__checklist-item-status {
+.launchpad__checklist-item-checkmark-container {
 	margin-right: 8px;
 	width: 20px;
-	.gridicon {
+
+	.launchpad__checklist-item-checkmark {
 		top: 2px;
 		fill: var( --studio-gray-50 );
+		vertical-align: text-bottom;
 	}
 }
 
-.launchpad__checklist-item-status-pending {
-	border: 1px solid var( --studio-gray-10 );
-	// stylelint-disable-next-line declaration-property-unit-allowed-list
-	border-radius: 50%;
-	box-sizing: border-box;
-	display: block;
-	height: 12px;
-	margin: 5px;
-	transition: all 0.1s;
-	width: 12px;
-}
-
-.launchpad__checklist-item-status-complete {
-	fill: var( --color-success );
-	vertical-align: text-bottom;
-}
-
+// general styles
 .launchpad__checklist-item-text {
 	flex-grow: 1;
 	font-size: $font-body;
 	line-height: 20px;
 	text-align: left;
 	font-weight: 600;
-	&.is-complete {
-		opacity: 0.8;
+	// this line overrides the hover state so it needs to be set to inherit
+	color: var( --studio-gray-50 );
```

Comment:
Just to make sure, we'd like to change checklist item text color for _incomplete_ tasks as well?

Currently, tasks that haven't been finished will be `--studio-gray-50` too. If so, then feel free to disregard this comment.